### PR TITLE
Headset mousedrop like PDA

### PR DESCRIPTION
**Pull request by jacklecroy**
## About The Pull Request

Drag a headset onto you to open the frequency menu.

## Why It's Good For The Game

Easier to adjust frequencies without having to take off the headset. Useful for traitors with uplink in headset.

## Changelog
:cl: That REALLY Good Soda Flavor
qol: You can drag a headset onto yourself to open the menu.
/:cl:

### CODE REVIEW

File: code/game/objects/items/devices/radio/headset.dm [hunk 1]
```diff
@@ -79,6 +79,12 @@ GLOBAL_LIST_INIT(channel_tokens, list(
 	. = ..()
 	.["headset"] = TRUE
 
+/obj/item/radio/headset/MouseDrop(mob/over, src_location, over_location)
+	var/mob/M = usr
```
Comment (tralezab) [inline code/game/objects/items/devices/radio/headset.dm:83]: use descriptive var names, please

File: code/game/objects/items/devices/radio/headset.dm [hunk 2]
```diff
@@ -79,6 +79,12 @@ GLOBAL_LIST_INIT(channel_tokens, list(
...
+	if((M == over) && usr.canUseTopic(src, BE_CLOSE, FALSE, NO_TK))
```
Comment (tralezab) [inline code/game/objects/items/devices/radio/headset.dm:84]: if you're going to typecast usr as a mob in line 83, then you should be calling canUseTopic on the typecasted var in 84

**Review by tralezab — DISMISSED** [review]
boop!

**Comment by datorangebottle** [comment]
I honestly didn't know PDA mousedrop existed until I saw this PR.

**Comment by UnokiAs** [comment]
Could be cool if you could adjust your headset frequency while wearing it with alt click having to put it in hands every time as traitor is annoying

**Comment by jacklecroy** [comment]
> 
> 
> ~Could be cool if you could adjust your headset frequency while wearing it with alt click having to put it in hands every time as traitor is annoying~ Wait I'm dumb I didn't read the pr correctly

I think alt click is used to toggle high volume mode.

**Comment by optimumtact** [comment]
an actual quality of life PR

**Comment by GoldenAlpharex** [comment]
I think the tweak: tag was removed and turned into qol:, no?

**Comment by iwishforducks** [comment]
Random thought: Are you able to change headset settings while in crit, allowing you to turn on hotmic and thus call for help?

**Comment by ShizCalev** [comment]
@iwishforducks Already handled by canusetopic / TRAIT_UI_BLOCKED

**Pull request merged**